### ISSUE TITLE
VSCode: "Getting Started" walkthrough text appears cut-off at small widths

### ISSUE THREAD

**Issue by kevinushey** [quarto-dev/quarto#72]
See the attached screenshot; the "Edit an equation" text appears cut-off in the middle of the page:

<img width="1488" alt="Screenshot 2023-02-06 at 1 29 03 PM" src="https://user-images.githubusercontent.com/1976582/217092092-40966f9c-ee98-45ee-877f-dde689a81eb7.png">

Comparing with other walkthroughs, I think the confusion arises because other examples always use an accompanying image, and the walkthrough resizes the image so that it displays either beside or below the walkthrough content...

<img width="1489" alt="Screenshot 2023-02-06 at 1 32 01 PM" src="https://user-images.githubusercontent.com/1976582/217092645-ce303e8c-248b-4c33-8833-4e59c55b4483.png">

<img width="666" alt="Screenshot 2023-02-06 at 1 32 12 PM" src="https://user-images.githubusercontent.com/1976582/217092662-60b225a7-30c5-4ea1-96c1-e3e051a209b0.png">

Minor, but it is a bit confusing at first glance.

**Comment by kevinushey** [comment]
Perhaps related is that you end up with a lot of empty space at certain widths as well.

<img width="849" alt="Screenshot 2023-02-06 at 1 36 12 PM" src="https://user-images.githubusercontent.com/1976582/217093307-9ad6388d-e5b4-4cae-ad48-0a3a77c3e7f5.png">

**Comment by jjallaire** [comment]
Yeah, I agree this is awkward (although I also think that requiring art for a walkthrough is a bit much which is probably why I didn't commission any!).
